CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,8 @@
+Add: /iot/configGroups API endpoints (as equivalent to /iot/services) (#752)
+Refresh Documentation
+Add NGSIv2 metadata support to device provisioned attributes
+Fix: Error message when sending measures with unknown/undefined attribute
+Add Null check within executeWithSecurity() to avoid crash (#829)
+Add NGSIv2 metadata support to attributeAlias plugin.
+Add mongodb authentication: IOTA_MONGO_USER, IOTA_MONGO_PASSWORD and IOTA_MONGO_AUTH_SOURCE (#844)
```

Comment:
CHANGES_NEXT_RELEASE changes show the addition of 7 lines. Are all them due to your PR? Please, reduce the added lines to just the one corresponding to your contribution, which I understand is:

> Add: /iot/configGroups API endpoints (as equivalent to /iot/services) (#752)

Reply:
Resolved by https://github.com/telefonicaid/iotagent-node-lib/pull/780/commits/dacb188a03c3c0063c88e9eb488d6a9f95825ff5. But it has created a conflict with master.